models/index.js

```diff
@@ -0,0 +1,28 @@
+const Sequelize = require('sequelize')
+const authorsModel = require('./authors')
+const genresModel = require('./genres')
+const novelsModel = require('./novels')
+const novelsGenresModel = require('./novelsGenres')
+
+const connection = new Sequelize('greatNovels', 'greatNovels', 'n0^3l$', {
+  host: 'localhost', dialect: 'mysql'
+})
+
+const authors = authorsModel(connection, Sequelize)
+const genres = genresModel(connection, Sequelize)
+const novels = novelsModel(connection, Sequelize, authors)
+const novelsGenres = novelsGenresModel(connection, Sequelize, novels, genres)
+
+novels.belongsTo(authors)
+authors.hasMany(novels)
+// authors.hasMany(genres)
```

Comment:
Always deleted commented out code before finalizing, this stuff ends up cluttering master in the real world